### PR TITLE
Wrap text in TextViewer

### PR DESCRIPTION
**Pull request by robert00s**
Minor fix.
Wrap text in title TextViewer widget.

Before:
screenshot 2017-07-30 17-32-16

After:

screenshot 2017-07-30 17-28-14

### CODE REVIEW

File: frontend/ui/widget/textviewer.lua [hunk 1]
```diff
@@ -80,11 +80,11 @@ function TextViewer:init()
         }
     end
 
-    local title_text = TextWidget:new{
+    local title_text = TextBoxWidget:new{
         text = self.title,
         face = self.title_face,
         bold = true,
-        width = self.width - 2*self.title_padding - 2*self.title_margin,
+        width = self.width * 0.9,
```
Comment (Frenzie) [inline frontend/ui/widget/textviewer.lua]: What's the reason behind this change?

Comment (robert00s) [inline frontend/ui/widget/textviewer.lua]: Sometimes close button (X) overwrites title text.

Comment (Frenzie) [inline frontend/ui/widget/textviewer.lua]: In that case couldn't you do something like a `getSize().w` on it?

Comment (Frenzie) [inline frontend/ui/widget/textviewer.lua]: I just realized I might've been a bit cryptic. I mean something like

```lua
width = self.width - 2*self.title_padding - 2*self.title_margin - close_button_width,
```

where you'd get the CloseButton width by calling getSize().w on a CloseButton object.

**Comment by Frenzie** [comment]
Awesome, thanks! Mystery one-off values like 0.9 should be avoided when possible. :-)
